blender/arm/make_renderpath.py

```diff
@@ -283,7 +308,6 @@ def build():
             assets.add_shader_pass('blur_gaus_pass')
 
         if rpdat.rp_ssr:
-            wrd.world_defs += '_SSR'
```

Comment:
What's the deal with removing this line?

Reply:
I added that myself and it got included in one of the pull requests.
I'm pretty sure, verified on older version and this line's not present.

Reply:
> I added that myself and it got included in one of the pull requests

Hm, can you please point to the pull request where it got included?

Reply:
It's from the water pass changes, since we need to know whether we'r using ssr or not.
Fixed it !